client/src/routes/CreateSessionModal/components/DateTimePicker.tsx

```diff
@@ -136,36 +104,47 @@ const Picker: React.FC<PickerProps> = ({
       <Wrapper>
         <Row
           onPress={() => {
-            setMode('date');
-            setShowPicker(true);
+            setShowTimePicker(false);
+            setShowDatePicker(!showDatePicker);
           }}>
           <Body16>
             <BodyBold>{t('date')}</BodyBold>
           </Body16>
-          <Body16>{selectedDate.local().format('dddd, D MMM')}</Body16>
+          <SelectedText isActive={showDatePicker}>
+            {selectedDate.local().format('dddd, D MMM')}
+          </SelectedText>
         </Row>
+        {showDatePicker && (
+          <DateTimePicker
+            mode="date"
+            selectedValue={selectedDate}
+            setValue={setSelectedDate}
+            close={() => setShowDatePicker(false)}
+            minimumDate={minimumDate}
+            maximumDate={maximumDate}
+          />
+        )}
         <Row
           onPress={() => {
-            setMode('time');
-            setShowPicker(true);
+            setShowDatePicker(false);
+            setShowTimePicker(!showTimePicker);
```

Comment:
Hot tip! 💡  There's a [`useBottomSheet`](https://gorhom.github.io/react-native-bottom-sheet/hooks/#usebottomsheet) hook that exposes an [`expand`](https://gorhom.github.io/react-native-bottom-sheet/methods#expand) function to expand the modal to maximum height :)

Reply:
[5a01269](https://github.com/29ki/29k/pull/1018/commits/5a01269ea404009310457efdc98b5e646df5944c)